iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/SV.java

```diff
@@ -262,8 +235,21 @@ private boolean isAllDataForCalculatingOtherSide() {
         return !(Double.isNaN(p) || Double.isNaN(q) || Double.isNaN(u) || Double.isNaN(a));
     }
 
-    private boolean isAllDataForCalculatingOtherSideDcApproximation() {
-        return !(Double.isNaN(p) || Double.isNaN(a));
+    private boolean isAllDataForCalculatingOtherSideDcApproximation(double zb) {
+        return !(Double.isNaN(p) || Double.isNaN(a) || Double.isNaN(zb));
+    }
+
+    private SV otherSide(double r, double x, double g1, double b1, double g2, double b2, double rho, double alpha, double zb) {
```

Comment:
maybe we could rename the normalization parameter to v1v2 ?

Reply:
Done. I changed by zbase.